runtime/src/main/java/org/corfudb/runtime/CompactorMetadataTables.java

```diff
@@ -20,12 +20,12 @@ public class CompactorMetadataTables {
     private Table<StringKey, CheckpointingStatus, Message> compactionManagerTable;
     private Table<TableName, CheckpointingStatus, Message> checkpointingStatusTable;
     private Table<TableName, ActiveCPStreamMsg, Message> activeCheckpointsTable;
-    private Table<StringKey, RpcCommon.TokenMsg, Message> checkpointTable;
+    private Table<StringKey, RpcCommon.TokenMsg, Message> compactorUtilsTable;
 
     public static final String COMPACTION_MANAGER_TABLE_NAME = "CompactionManagerTable";
     public static final String CHECKPOINT_STATUS_TABLE_NAME = "CheckpointStatusTable";
     public static final String ACTIVE_CHECKPOINTS_TABLE_NAME = "ActiveCheckpointsTable";
-    public static final String CHECKPOINT_TABLE_NAME = "checkpoint";
+    public static final String COMPACTOR_UTILS_TABLE = "CompactorUtilsTable";
```

Comment:
Looking at these names, I think `CompactionManagerTable` and `CompactorUtilsTable` are not as clear and straightforward as the other two table names. Based on the usages, `CompactionManagerTable` might be better if we call it `CycleStatusTable`. 'CompactorUtilsTable' is used to store compaction controlling/triggering settings and  the minToken to trim. I don't have a better idea but I think 'Utils' is ambiguous. Down the road the compactor will have new features that require to persist more settings/information and I think the meaning of each table should be made more clear.

Reply:
Hmm... I think changing the name of Manager to CycleStatusTable seems a good idea. 
What about CompactionControlsTable instead of UtilsTable (don't feel this is the best name but couldn't think of any)?

Reply:
`CompactionCycleStatusTable` just to be clear. And yes `CompactionControlsTable` is better. Thanks!

Reply:
The problem with changing CompactionManagerStatus is that, there are too many dependencies to it. 
Like everywhere when it's read it's referred to as managerStatus. 
And also something like this - livenessValidator.shouldChangeManagerStatus()